source/extensions/common/wasm/wasm.cc

```diff
@@ -1,6 +1,6 @@
 #include "extensions/common/wasm/wasm.h"
 
-#include <stdio.h>
+#include <cstdio>
```

Comment:
Why do we need stdio here, do we print stdout directly?

Reply:
These are left over from debugging.  YCM doesn't seem to report unnecessary includes, but I checked and I can remove them.